scripts/mk_all.sh

```diff
@@ -7,8 +7,18 @@ for d in $(find $DIR/../src/ -type d)
 # For every subfolder of src/ (including src/ itself)
 do
   cd "$d" # cd to that directory
+  echo /- automatically generated file importing all files in this directory and subdirectories. -/ > all.lean
```

Comment:
```suggestion
  echo "/- automatically generated file importing all files in this directory and subdirectories. -/" > all.lean
```
I guess it's technically speaking not necessary... but ... whatever